dashboard/lib/announcements.rb

```diff
@@ -23,7 +23,7 @@ def self.get_announcement_for_page(page)
     banner = banners[banner_id_for_page]
 
     #Returns banner if no DCDO property or if DCDO property is present and set to true
```

Comment:
Nit: Would you mind updating the comment here to reflect the updated logic please?

Reply:
Ah yes, good catch!